static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -84,6 +86,26 @@ define([
         var ophanUrl = 'https://dashboard.ophan.co.uk/graph/breakdown?ab=' + this.config.test.id;
         this.getElem('ophan').innerHTML = '<a href="' + ophanUrl + '"">graph</a>';
 
+        if (window.abConfigSwitches.abShowHypothesesDashboard) {
+            var hypothesis;
+            var criteria = this.config.test.audienceCriteria ? true : false;
+            var description = this.config.test.description ? true : false;
+            var reason = this.config.test.reason ? true : false;
+            var success = this.config.test.successMeasure ? true : false;
+            var outcome = this.config.test.idealOutcome ? true : false;
```

Comment:
Not quite. They're checking that each property exists. If so we get a true, otherwise false. Then I can say "if all these things exist, I can build the hypothesis statement".